package.json

```diff
@@ -35,19 +35,21 @@
     "install-if-deps-outdated": "node bin/install-if-deps-outdated.js",
     "lint": "npm run lint:js && npm run lint:css",
     "lint:css": "stylelint 'client/**/*.scss' 'client/**/*.css' 'assets/**/*.scss' 'assets/**/*.css'",
-    "lint:css-fix": "npm run lint:css -- --fix",
     "lint:js": "eslint . --ext=js,jsx",
     "lint:php": "./vendor/bin/phpcs --standard=phpcs.xml.dist $(git ls-files | grep .php$)",
-    "lint:php-fix": "./vendor/bin/phpbf --standard=phpcs.xml.dist $(git ls-files | grep .php$)"
+    "lint:php-fix": "./vendor/bin/phpbf --standard=phpcs.xml.dist $(git ls-files | grep .php$)",
+    "format": "npm run format:js && npm run format:css",
+    "format:js": "npm run format:provided 'assets/**/*.js' 'client/**/*.js'",
+    "format:css": "npm run format:provided 'client/**/*.scss' 'assets/**/*.css'",
+    "format:provided": "prettier --write"
   },
   "husky": {
     "hooks": {
       "pre-commit": "lint-staged"
     }
   },
   "lint-staged": {
-    "*.{js,jsx}": "npm run lint:js",
-    "*.scss": "npm run lint:css",
+    "*.{js,jsx,scss,css}": "npm run format:provided",
```

Comment:
I'm still concerned with this change. It's basically disables linter checks and relies on formatter only. 
Side-effect - linter scripts (lint:js, lint:css) seem to be not used anymore.

The other outcome is if you just remove a single space in the current file e.g. `client/deposits/details/index.js` and commit the change. The committed diff will be bloated with other formatting changes introduced by prettier which you didn't staged.

As I said before, this is my personal opinion on the topic and I wouldn't like forcing it. I'd ask someone else here for another opinion.

Reply:
LOL, we reached the same conclusions almost at the same time.

> I'm still concerned with this change. It's basically disables linter checks and relies on formatter only.
Side-effect - linter scripts (lint:js, lint:css) seem to be not used anymore.

https://github.com/Automattic/woocommerce-payments/pull/631#discussion_r456360981

> The committed diff will be bloated with other formatting changes introduced by prettier which you didn't staged.

https://github.com/Automattic/woocommerce-payments/pull/631#pullrequestreview-450551039

Reply:
I think this has been resolved in https://github.com/Automattic/woocommerce-payments/pull/631#issuecomment-662374574, what do you think?